Tests/UnitTests/Misc/SystemInfoTests.swift

```diff
@@ -57,18 +57,6 @@ class SystemInfoTests: TestCase {
         expect(systemInfo.observerMode) == !finishTransactions
     }
 
-    func testIsSandbox() throws {
```

Comment:
I removed these tests so we don't need to also inject `isRunningInSimulator` to `SystemInfo`.
Sandbox environment is detected through `SandboxEnvironmentDetector`, so these tests were duplicated.